remix.init/netlify-toml

```diff
@@ -2,6 +2,10 @@
 command = "remix build"
 publish = "public"
 
+[dev]
```

Comment:
This should be getting picked up from the defaults for Remix, but I'll investigate that later.